avaframe/ana3AIMEC/ana3AIMECCfg.ini

```diff
@@ -24,6 +24,9 @@ resTypes = ppr|pft|pfv
 # data result type for runout analysis (ppr, pft, pfv)
 runoutResType = ppr
 
+# show cross profile max or mean values on slComparisonstat plot
```

Comment:
please indicate what the options are, i.e. how the parameter needs to be set to e.g. get the mean values...

Reply:
done